templates/single-product/related.php

```diff
@@ -22,9 +22,12 @@
 if ( $related_products ) : ?>
 
 	<section class="related products">
-
-		<h2><?php esc_html_e( 'Related products', 'woocommerce' ); ?></h2>
-
+        <?php
+        $heading = apply_filters( 'woocommerce_product_related_heading', __( 'Related products', 'woocommerce' ) );
```

Comment:
Could you please change the filter name to `woocommerce_product_related_products_heading` instead?

Reply:
Done.

Best Regards,
Andrej